piet-svg/src/lib.rs

```diff
@@ -272,9 +272,17 @@ impl piet::RenderContext for RenderContext {
             .unwrap()
             .insert(layout.font_face.clone());
 
+        // We use the top of the text for y position, but SVG uses baseline, so we need to convert
+        // between the two.
+        //
+        // `dominant-baseline` gets us most of the way (to the top of the ascender), so we add a
+        // small fiddle factor in to cover the difference between the top of the line and the top
+        // of the ascender (currently 6% of the font height, calcuated by eye).
+        let y = pos.y + 0.06 * layout.size().height;
```

Comment:
I generally find that fudge factors like this are a sign that the implementation isn't *quite* doing the right thing. That said, I had a skim through SVG docs and don't see any better way to do this, so I'll approve.